app_legacy/Helpers/render/game.php

```diff
@@ -21,11 +21,21 @@ function gameAvatar(
         if ($label !== false) {
             $title = $game['GameTitle'] ?? $game['Title'] ?? null;
             $consoleName = $game['Console'] ?? $game['ConsoleName'] ?? null;
+            sanitize_outputs($title);   // sanitize before rendering HTML
+
+            $flexDirection = $iconSize >= 32 ? 'col' : 'row';
```

Comment:
This refactor fixes a bug and seems a bit easier to reason about:

```php
$flexDirection = $iconSize >= 32 ? 'flex-col' : 'flex-row';
$alignItems = $flexDirection === 'flex-col' ? 'items-start' : 'items-center';

$consoleDiv = '';
if ($consoleName) {
    $consoleClassNames = 'console-name';
    if ($flexDirection === 'flex-row') {
        $consoleClassNames .= " mx-0.5 border-l-2 border-white/[.25] pl-1.5";
    }
    $consoleDiv = "<div class='$consoleClassNames'>$consoleName</div>";
}

$gameTitle = renderGameTitle($title);

$label = <<<HTML
    <div class='flex $flexDirection $alignItems gap-1'>
        $gameTitle
        $consoleDiv
    </div>
HTML;
```

Tailwind tokenizes by the entire name of the string being present in a file. It isn't able to detect "col", "or", "start", or "center". If the page doesn't already have those class names elsewhere, the game title will be broken.